requirements.txt

```diff
@@ -1,9 +1,11 @@
-# Core modules (mandatory)
-jupyterlab~=4.2.5
```

Comment:
Is the removal of `jupyterlab` intentional?

I guess if we don't specify a version of `jupyterlab` we'll get the latest, modulus [the version required by jupyterlite](https://github.com/jupyterlite/jupyterlite/blob/main/requirements-build.txt#L10C1-L10C24) which makes sense.

Reply:
Yes, this is intentional indeed, as it should be better to pull the version of JupyterLab that `jupyterlite-core` pulls in.